data/security_monitoring/data.yaml

```diff
@@ -43,6 +43,10 @@ image_map:
       - guardduty
     image: amazon_guardduty
     append: false
+  - source:
+      - route53
```

Comment:
I would suggest to move Route53 to the Amazon source array with the other images, line ~28.

Reply:
Thanks! I totally missed that and i've added something in there before 🤦